modules/sql-engine/src/integrationTest/java/org/apache/ignite/internal/sql/engine/ItComputeSystemViewTest.java

```diff
@@ -0,0 +1,309 @@
+    @Override
+    protected int initialNodes() {
+        return 2;
+    }
+
+    @ParameterizedTest
+    @ValueSource(booleans = {true, false})
+    public void checkMeta(boolean isClient) {
+        String query = "SELECT * FROM SYSTEM.COMPUTE_TASKS";
+
+        Ignite entryNode = isClient ? IgniteClient.builder().addresses("localhost").build() : CLUSTER.node(0);
+        Ignite targetNode = CLUSTER.node(0);
+
+        JobDescriptor<Void, Void> job = JobDescriptor.builder(InfiniteJob.class).units(List.of()).build();
+        JobExecution<Void> execution = entryNode.compute().submit(JobTarget.node(clusterNode(targetNode)), job, null);
+
+        await().until(execution::stateAsync, willBe(jobStateWithStatus(EXECUTING)));
+
+        try {
+            // Verify metadata.
+            assertQuery(query)
+                    .withDefaultSchema("SYSTEM")
+                    .columnMetadata(
+                            new MetadataMatcher().name("COORDINATOR_NODE_ID").type(ColumnType.STRING).nullable(false),
+                            new MetadataMatcher().name("ID").type(ColumnType.STRING).precision(36).nullable(true),
+                            new MetadataMatcher().name("STATUS").type(ColumnType.STRING).nullable(true),
+                            new MetadataMatcher().name("CREATE_TIME").type(ColumnType.TIMESTAMP).nullable(true),
+                            new MetadataMatcher().name("START_TIME").type(ColumnType.TIMESTAMP).nullable(true),
+                            new MetadataMatcher().name("FINISH_TIME").type(ColumnType.TIMESTAMP).nullable(true)
+                    )
+                    .check();
+        } finally {
+            execution.cancelAsync();
+
+            await().until(execution::stateAsync, willBe(jobStateWithStatus(CANCELED)));
+
+            closeQuiet(entryNode);
+        }
+    }
+
+    @ParameterizedTest
+    @ValueSource(booleans = {true, false})
+    void viewRunningJobs(boolean isClient) {
+        Ignite entryNode = isClient ? IgniteClient.builder().addresses("localhost").build() : CLUSTER.node(0);
+        Ignite targetNode = CLUSTER.node(0);
+
+        try {
+            ClockService clockService = unwrapIgniteImpl(targetNode).clockService();
+
+            long tsBefore = clockService.now().getPhysical();
+
+            JobDescriptor<Void, Void> job = JobDescriptor.builder(InfiniteJob.class).units(List.of()).build();
+            JobExecution<Void> execution = entryNode.compute().submit(JobTarget.node(clusterNode(targetNode)), job, null);
+
+            await().until(execution::stateAsync, willBe(jobStateWithStatus(EXECUTING)));
+
+            long tsAfter = clockService.now().getPhysical();
+
+            String query = "SELECT * FROM SYSTEM.COMPUTE_TASKS WHERE STATUS = ?";
+
+            List<List<Object>> res = sql(0, query, EXECUTING.name());
+
+            verifyQueryInfo(res.get(0), List.of(targetNode.name()), EXECUTING.name(), tsBefore, tsAfter);
```

Comment:
why not just check ROW data against  STATE of execution?
I mean 
```
verifyQueryInfo(res.get(0), await(execution.stateAsync()));
```

Reply:
this check is common for different compute api approaches, i don`t like idea do adopt it for such methods as :
`Map<ClusterNode, JobExecution<Void>> execution = entryNode.compute().submitBroadcast`